java/src/main/java/org/rocksdb/Transaction.java

```diff
@@ -1788,7 +1788,7 @@ public long getId() {
     AWAITING_PREPARE((byte)1),
     PREPARED((byte)2),
     AWAITING_COMMIT((byte)3),
-    COMMITED((byte)4),
+    COMMITTED((byte)4),
```

Comment:
This is also a public API, so let's do our best to keep the old name as an alias for the new name. It's not 100% possible for Java enums, but close enough, IMHO: https://stackoverflow.com/a/37092410/454544

Reply:
Added